currencies.json

```diff
@@ -122,6 +122,7 @@
   {
     "name": "@elastic/elasticsearch",
     "policy": "45-days",
+    "versions": ["7.9.0", "7.17.0", "8.15.0"],
```

Comment:
> "versions": ">= 7"

If we use this notation in the currencies.json, we need to initially generate the currencies.json the following:

```json
{
   "name": "@elastic/elasticsearch",
   "supportedVersions": ">= 7"
   "testedVersions": ["7.9.0", "8.15.0"]
   ...
```

In the test we use:

> CURRENCY.testedVersions.forEach(...)

The currency bot in the night still runs a major and minor update check. If there is a new version, a PR is created for the currencies.json diff.

e.g. for major:
```json
{
   "name": "@elastic/elasticsearch",
   "supportedVersions": ">= 7"
   "testedVersions": ["7.9.0", "8.15.0", "9.0.0"]
   ...
```

e.g. for minor 
```json
{
   "name": "@elastic/elasticsearch",
   "supportedVersions": ">= 7"
   "testedVersions": ["7.9.0", "8.16.0"]
   ...
```

This gives us **visibility**.
As soon as we merge the PR for the currencies.json, the testedVersions array is updated and the tests run against v9 or 8.16.0 or whatever.

>    "supportedVersions": ">= 7"

This can be publicly displayed and is the base how to generate the testedVersions array.